recipes/mkdocs-graphviz/meta.yaml

```
@@ -0,0 +1,42 @@
+{% set name = "mkdocs-graphviz" %}
+{% set version = "1.4.4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/mkdocs_graphviz-{{ version }}.tar.gz
+  sha256: 47586556ef2e137a2d58366ea9848a53c8051887c18e426e004d58ce551a0afa
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - markdown >=2.3.1
+    - python >=3.6
+
+test:
+  imports:
+    - mkdocs_graphviz
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://gitlab.com/rodrigo.schwencke/mkdocs-graphviz.git
+  summary: Render Graphviz graphs in Mkdocs, as inline SVGs and PNGs, natively compatible with Mkdocs Light & Dark Themes, directly from your Markdown (python3 version)
```

This is a bit long for a summary, it should generally be kept to less than ~80 characters.
```suggestion
  summary: Render Graphviz graphs in Mkdocs directly from your Markdown
```

---

Thanks. Sounds good. This was merged.